extensions/2.0/Vendor/MSFT_RigidBodies/schema/node.MSFT_physics.schema.json

```diff
@@ -0,0 +1,25 @@
+{
+    "$schema" : "http://json-schema.org/draft-04/schema",
+    "title" : "MSFT_RigidBodies glTF extension",
+    "type" : "object",
+    "description" : "Physics properties for a node.",
+    "allOf" : [ { "$ref" : "glTFChildOfRootProperty.schema.json" } ],
+    "properties" : {
+        "rigidBody" : {
+            "allOf" : [ { "$ref" : "node.MSFT_RigidBodies.rigidBody.schema.json" } ],
+            "description" : "A set of parameter values that enable this node and its children to be moved by physics simuation."
+        },
+        "physicsMaterial" : {
+            "allOf": [ { "$ref": "glTFid.schema.json" } ],
+            "description": "The index of the physics material in the top level physicsMaterials array."
+        },
+        "collider" : {
```

Comment:
Isn't `collider` already specified in `node.MSFT_CollisionPrimitives.schema.json`? Or is this a different collider?

Reply:
This is the collider used for physics simulation. MSFT_collision_primitives is explicitly broken into a separate schema to allow other use-cases; referencing the collider from the MSFT_rigid_bodies extension allows you to use different collider representations for different extensions which may have different requirements (consider a node which is both a UI widget and a physically simulated body).